ZenPacks/zenoss/OpenStackInfrastructure/events.py

```diff
@@ -718,6 +718,10 @@ def volume_update(device, dmd, evt):
     elif 'detach.end' in evt.eventClassKey:
         # the volume is being detached from an instance
         setattr(objmap, 'set_instance', '')
+    # make sure objmap has volume_type and the len == uuid len
+    if hasattr(objmap, 'volume_type') and len(objmap.volume_type) == 36:
```

Comment:
Its probably better to try to detect a UUID via Regex here instead of raw lenght.